pkg/plugin/plugin.go

```diff
@@ -387,26 +387,88 @@ func ScanRootForDevices(nxGzipEnabled bool) ([]string, error) {
 	// relies on GHW_CHROOT=/host/dev
 	// lsblk -f --json --paths -s | jq -r '.blockdevices[] | select(.fstype != "xfs")' | grep mpath | grep -v fstype | sort -u | wc -l
 	// This may be the best way to get the devices.
+	config, err := loadDevicePluginConfig()
+	if err != nil {
+		klog.Warningf("ScanRootForDevices: failed to load config, proceeding with default behavior: %v", err)
+		config = &api.DevicePluginConfig{}
+	}
+
 	block, err := ghw.Block()
 	if err != nil {
 		fmt.Printf("Error getting block storage info: %v", err)
 		return nil, err
 	}
 	devices := []string{}
 	fmt.Printf("DEVICE: %v\n", block)
+
 	for _, disk := range block.Disks {
 		fmt.Printf("    - DISK: %v\n", disk.Name)
 		for _, part := range disk.Partitions {
 			fmt.Printf("        - PART: %v\n", part.Disk.Name)
-			devices = append(devices, part.Name)
+			devices = append(devices, "/dev/"+part.Name)
 		}
-		devices = append(devices, disk.Name)
+		devices = append(devices, "/dev/"+disk.Name)
 	}
+
 	if nxGzipEnabled {
-		devices = append(devices, nxGzipDevicePath)
+		devices = append(devices, "/dev/"+nxGzipDevicePath)
```

Comment:
```suggestion
		devices = append(devices, "/dev/crypto/nx-gzip")
```
Let's make this a constant